contracts/cw1155-base/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "cw1155-base"
+version = "0.6.0-alpha1"
```

Comment:
I think I just bumped this up to `-alpha3` trying to get multitest working for another app I'm building.

Reply:
See #257 and #258

Reply:
fixed now.